components/post_view/message_attachments/message_attachment/message_attachment.jsx

```diff
@@ -198,7 +198,14 @@ export default class MessageAttachment extends React.PureComponent {
                     key={'attachment__field-caption-' + i + '__' + nrTables}
                     width='50%'
                 >
-                    {field.title}
+                    <Markdown
+                        message={field.title}
+                        options={{
```

Comment:
we should make this options a constant to avoid passing a new object every time.

also I do not see where `mentionHighlight` or `markdown` options are used

Reply:
@enahum, I removed mentionHIghlight and autolinkedUrlSchemes because they were unnecessary.
Im not sure what I did with the options is what you meant in your comment...